samples/sensor/bh1749/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_LOG=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_HAS_SEGGER_RTT=y
```

Comment:
This is a hardware property, so this option should be enabled by a board, not by an application (unless it is strictly associated with some hardware featuring a JLink debug interface). I think you should remove the RTT related options from this file.